src/hooks/__tests__/use-counter.test.ts

```diff
@@ -1,9 +1,11 @@
-import { renderHook } from '@testing-library/react-hooks';
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
can you tell me why this pragma is necessary? cannot it be configured instead of adding this in all test files?

Reply:
Fixed now. Switched test-environment from 'node' to 'jsdom' globally